notebooks/DM-31046-Zooniverse-cutouts.ipynb

```diff
@@ -0,0 +1,700 @@
+      "text/plain": [
+       "<IPython.core.display.Image object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "files = glob.glob(os.path.join(config.outputPath, \"images\", \"*.png\"))\n",
+    "for file in files[:10]:\n",
+    "    image = IPython.display.Image(file)\n",
+    "    IPython.display.display(image)"
```

Comment:
I'm not sure I understand how this leads to the plot that's shown. Is each PNG file a pre-arranged row of three images?

Reply:
Yes, the task makes unified template/science/difference cutouts for use in Zooniverse projects.